chrome/content/zotero/xpcom/storage/storageLocal.js

```diff
@@ -563,6 +568,32 @@ Zotero.Sync.Storage.Local = {
 			}
 		);
 	},
+
+
+	/**
+	 * Touch all the given items with a new `lastAccessed` timestamp
+	 *
+	 * @param {Zotero.Item[]} items
+	 * @return {Promise}
+	 */
+	_updateLastAccessed: async function (items) {
+		let timestamp = Date.now();
+
+		await Zotero.Utilities.Internal.forEachChunkAsync(
+			items,
+			1000,
```

Comment:
This should actually be set to `Zotero.DB.MAX_BOUND_PARAMETERS` (which is 999), as should `updateSyncStates()`.